### DISCUSSION TITLE
边是否可以支持随意拉拽？

### DISCUSSION THREAD

**Discussion by gushenyi123** [antvis/X6#2426] [Q&A]
边是否可以支持随意拉拽？如果可以的话，麻烦给个指导信息，谢谢

**Comment by zmcode** [comment]
https://antv-x6.gitee.io/zh/docs/api/registry/edge-tool/#vertices

**Comment by No54** [comment]
如何给Edge加上基本选项？ 就像graph一样？ 很多情况是手动加的Edge，而不是通过 addEdge出来的

**Comment by NewByVector** [comment]
@gushenyi123 可以使用这个[工具](https://antv-x6.gitee.io/zh/docs/api/registry/edge-tool/#source-arrowhead-%E5%92%8C-target-arrowhead)。
